ericberringer/currency-kidz-react
# Ck savings goal

Type of change:

Testing Instructions

On profile, if the savings goal amount is zero the user should have a form to enter a savings goal. Once the goal is entered a progress bar should be there with total saved so far,  goal amount, and percentage of progress.

git fetch --all 
git checkout CK-SavingsGoal

Checklist:

X(I have preformed a self review)

X(Code follows style guidelines of project)

(I have commented my code. Especially w/ hard to understand code)

X(Changes I have made generate no new warnings or errors)

X(I have added test instructions that prove my fix is effective or that my feature works)

(I am not ruminating about the things my therapist told me to forget about b/c I cannot do anything about them. For example, the state of the world. )

- Fixes #
